src/main/java/com/box/sdk/BoxTask.java

```diff
@@ -0,0 +1,281 @@
+package com.box.sdk;
+
+import java.net.URL;
+import java.text.ParseException;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+import com.eclipsesource.json.JsonArray;
+import com.eclipsesource.json.JsonObject;
+import com.eclipsesource.json.JsonValue;
+
+/**
+ * Represents a task on Box.
```

Comment:
Add a little more descriptive comment here:

Represents a task on a file. Tasks can have due date and can be assigned to specific user.